simple_salesforce/api.py

```diff
@@ -679,6 +713,12 @@ def __init__(
                                      object_name=object_name,
                                      sf_version=sf_version))
 
+    @property
+    def session_id(self):
```

Comment:
I use a property here to make it backward compatible with the tests already in place. However, if this officially gets merged in we'd probably want to just pass most everything through the `salesforce` parameter.